src/Compilers/Core/Portable/Optional.cs

```diff
@@ -56,5 +56,15 @@ public T Value
         {
             return new Optional<T>(value);
         }
+
+        /// <summary>
+        /// Returns a string representation of the optional value.
+        /// </summary>
+        public override string ToString()
+        {
+            return _hasValue
+                ? _value.ToString()
```

Comment:
If we wanted to be literal, I would use "no value" instead of "default". It matches the terminology used by the API itself.